website/docs/r/sfn_state_machine.html.markdown

```diff
@@ -108,7 +108,7 @@ The following arguments are supported:
 
 ### `logging_configuration` Configuration Block
 
-* `log_destination` - (Optional) Amazon Resource Name (ARN) of CloudWatch log group. Make sure the State Machine does have the right IAM Policies for Logging.
+* `log_destination` - (Optional) Amazon Resource Name (ARN) of CloudWatch log group. Make sure the State Machine does have the right IAM Policies for Logging. Log Group ARN must be provided with '*' qualifier.
```

Comment:
Hi @pradeepbhadani! Thanks for the contribution, I think we'd prefer if this qualifier in the documentation followed the AWS documentation as its a little clearer. Otherwise LGTM! 🚀 

https://docs.aws.amazon.com/step-functions/latest/apireference/API_CloudWatchLogsLogGroup.html#StepFunctions-Type-CloudWatchLogsLogGroup-logGroupArn

```suggestion
* `log_destination` - (Optional) Amazon Resource Name (ARN) of CloudWatch log group. Make sure the State Machine does have the right IAM Policies for Logging. The ARN must end with :*.
```

Reply:
@breathingdust  - Thanks for reviewing. updated doc as per suggestion.